Mathlib/GroupTheory/Index.lean

```diff
@@ -248,10 +250,13 @@ theorem dvd_index_map {f : G →* G'} (hf : f.ker ≤ H) :
   apply dvd_mul_right
 
 @[to_additive]
-theorem index_map_eq {f : G →* G'} (hf1 : Function.Surjective f)
-    (hf2 : f.ker ≤ H) : (H.map f).index = H.index :=
+theorem index_map_eq (hf1 : Surjective f) (hf2 : f.ker ≤ H) : (H.map f).index = H.index :=
   Nat.dvd_antisymm (H.index_map_dvd hf1) (H.dvd_index_map hf2)
 
+@[to_additive]
+lemma index_map_of_bijective (hf : Bijective f) (S : Subgroup G) : (S.map f).index = S.index :=
```

Comment:
```suggestion
lemma index_map_of_bijective (hf : Bijective f) : (H.map f).index = H.index :=
```
for consistency with the previous

Reply:
I've renamed the subgroup, but `H` should stay explicit in final position (consistency with the previous doesn't hold as the previous has an assumption explicitly mentioning `H`, which is not the case here)